lib/puppet/provider/package/pacman.rb

```diff
@@ -8,7 +8,6 @@
   This provider supports the `install_options` attribute, which allows command-line flags to be passed to pacman.
   These options should be specified as a string (e.g. '--flag'), a hash (e.g. {'--flag' => 'value'}),
   or an array where each element is either a string or a hash."
-
```

Comment:
Can you restore this line break so there's some separation between the doc comment and the `yaourt?` method?